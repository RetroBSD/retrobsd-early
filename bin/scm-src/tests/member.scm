(memq 'a '(a b c))
(memq 'b '(a b c))
(memq 'a '(b c d))
(memq (list 'a) '(b (a) c))
(member (list 'a) '(b (a) c))
(memv 101 '(100 101 102))

(define e '((a 1) (b 2) (c 3)))
(assq 'a e)
(assq 'b e)
(assq 'd e)
(assq (list 'a) '(((a)) ((b)) ((c))))
(assoc (list 'a) '(((a)) ((b)) ((c))))
(assv 5 '((2 3) (5 7) (11 13)))
