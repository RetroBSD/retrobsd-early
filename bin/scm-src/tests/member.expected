(memq 'a '(a b c)) --> (a b c)
(memq 'b '(a b c)) --> (b c)
(memq 'a '(b c d)) --> ()
(memq (list 'a) '(b (a) c)) --> ()
(member (list 'a) '(b (a) c)) --> ((a) c)
(memv 101 '(100 101 102)) --> (101 102)
(define e '((a 1) (b 2) (c 3))) --> ((a 1) (b 2) (c 3))
(assq 'a e) --> (a 1)
(assq 'b e) --> (b 2)
(assq 'd e) --> ()
(assq (list 'a) '(((a)) ((b)) ((c)))) --> ()
(assoc (list 'a) '(((a)) ((b)) ((c)))) --> ((a))
(assv 5 '((2 3) (5 7) (11 13))) --> (5 7)
